web/src/AddNeighbourhoodMode.svelte

```diff
@@ -449,4 +495,27 @@
     color: black;
     border: solid green 1px;
   }
+  .neighbourhood-boundary-summary {
+    border: dashed black 2px;
+    border-radius: 8px;
+    margin-bottom: 16px;
+    padding: 16px;
+  }
+  .step h3::before {
+    /* some content is necessary for the width to take effect. */
+    content: " ";
+    /* approximate width of widest emoji */
+    width: 24px;
+    display: inline-block;
+    margin-right: 8px;
+  }
+  .step.active h3::before {
+    content: "👉";
```

Comment:
Is it reasonable to use unicode/emoji chars like this on non-mac platforms?

I could explicitly include some icons from lucide, it just makes the logic a little more verbose.

Reply:
Works fine over here, I don't see why not?